test/__snapshots__/UniswapV3Staker.unit.spec.ts.snap

```diff
@@ -12,4 +12,8 @@ exports[`UniswapV3Staker.unit Stakes #claimReward has gas cost 1`] = `46922`;
 
 exports[`UniswapV3Staker.unit Stakes #stakeToken works and has gas cost 1`] = `116324`;
 
-exports[`UniswapV3Staker.unit Stakes #unstakeToken works and has gas cost 1`] = `100767`;
+exports[`UniswapV3Staker.unit Stakes #unstakeToken works and has gas cost 1`] = `100773`;
+
+exports[`UniswapV3Staker.unit Stakes #updateStake in a valid scenario has equivalent multicall operation gas cost 1`] = `165301`;
```

Comment:
iiuc, isn't it unfair to include the gas cost of claim in this equivalence test? because updateStake doesn't send out rewards. i'm worried that the actual gas difference might be smaller than ~36k, in which case i'm a little concerned about introducing redundancy

Reply:
just seeing this. good point. at first, it did include transfer. but now, in this case, claim would be taken out.